examples/game/breakout.rs

```diff
@@ -249,6 +249,7 @@ fn ball_collision_system(
                     Collision::Right => reflect_x = velocity.x < 0.0,
                     Collision::Top => reflect_y = velocity.y < 0.0,
                     Collision::Bottom => reflect_y = velocity.y > 0.0,
+                    Collision::Intersecting => panic!("Not supposed to happens"),
```

Comment:
Maybe in this case we should just do nothing? It will eventually either (1) phase through or (2) collide with the other side. I think thats probably preferable to picking an arbitrary side to bounce off of, panicking, or warning.

Reply:
I admit I was focused mainly on function `collide` and not in the game itself.

In the current breakout code the `Intersecting` case never happens, but could arise for example changing the ball velocity.
In that situation I would prefer my code to panic instead of silently ignore it, I think is more robust but is probably a question of style.

Reply:
Yeah style-wise I would prefer to just do nothing (and maybe leave a comment explaining the situation). I wouldn't put a panic here in a real game because the cost of crashing games is high and some amount of physics weirdness is to be expected. I also expect people to play around with the settings in this example / create new colliders / etc in a way that might allow this type of intersection to occur.